src/docs/content/observability/observability-scenario/pre-requisites/setup.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Setup"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+weight: 1
+tags:
+  - mushop setup
+---
+
+## Pre-requisites
```

Comment:
This prerequisites on the observability level, makes the visitors think that they need to do all these steps to be able to have basic observability, this is not true. These are optional steps for a specific use-case.

Reply:
this pre-requisite is under the observability-scenario